test-app/try-or-die.config.js

```diff
@@ -1,7 +1,8 @@
 /* eslint-disable n/no-missing-import */
 
-import { embroiderOptimized, embroiderSafe } from "@embroider/test-setup";
-import emberSourceChannelURL from "ember-source-channel-url";
+// import { embroiderOptimized, embroiderSafe } from "@embroider/test-setup";
+// import emberSourceChannelURL from "ember-source-channel-url";
+import { fileHandlers } from "../dist/index.js";
```

Comment:
```suggestion
import { fileOperations } from "../dist/index.js";
```

Maybe? Naming still feels a bit weird.

Reply:
As much as `fileOperations` sounds more accurate, `fileHandlers` gives a bit more "familiar" feeling. 🤷